interface_config.js

```diff
@@ -23,14 +23,26 @@ var interfaceConfig = { // eslint-disable-line no-unused-vars
     AUTHENTICATION_ENABLE: true,
     // the toolbar buttons line is intentionally left in one line, to be able
     // to easily override values or remove them using regex
-    MAIN_TOOLBAR_BUTTONS: ['microphone', 'camera', 'desktop', 'invite', 'fullscreen', 'hangup'], // jshint ignore:line
     /**
      * The index of the splitter button in the main toolbar. The splitter
      * button is a button in the toolbar that will be applied a special styling
      * visually dividing the toolbar buttons.
      */
     //MAIN_TOOLBAR_SPLITTER_INDEX: -1,
-    TOOLBAR_BUTTONS: ['profile', 'authentication', 'microphone', 'camera', 'desktop', 'recording', 'security', 'raisehand', 'chat', 'etherpad', 'sharedvideo', 'sip', 'dialpad', 'settings', 'hangup', 'filmstrip', 'contacts'], // jshint ignore:line
```

Comment:
Now we use "Share the link to the call" popup for security methods. So we didn't use 'security' here cause we don't have this button now